oauth2-server/src/main/java/com/clouway/oauth2/OAuth2ServletApiSupport.java

```diff
@@ -0,0 +1,14 @@
+package com.clouway.oauth2;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * @author Mihail Lesikov (mihail.lesikov@clouway.com)
+ */
+public interface OAuth2ServletApiSupport {
```

Comment:
Maby just OAuth2ApiSupport so that Servlet is not involved.

Reply:
sounds good to me

Reply:
It's related to Servlet API as method accepts HttpServletRequest and HttpServletResponse.

A better definition of this interface need to be provided which to not be related to the Servlet API. If it accepts friendly serve requests and responses then the client app which is using the library will be able to create adapter depending on the environment.

Reply:
It could happen in following PR btw.